<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hedm_calibration_options_dialog</class>
 <widget class="QDialog" name="hedm_calibration_options_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HEDM Calibration Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="view_grains_table">
     <property name="text">
      <string>Select Grains for Calibration Refinement</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="grid_layout">
     <item row="5" column="1">
      <widget class="QLabel" name="npdiv_label">
       <property name="text">
        <string>Number of Polar Subdivisions:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="num_grains_selected">
       <property name="text">
        <string>Number of grains to be refined:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>η and ω ranges can be edited within the &quot;Overlay Manager&quot;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QGroupBox" name="tolerances_group">
       <property name="title">
        <string>Tolerances</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="tolerances_selected_grain_label">
          <property name="text">
           <string>Selected Grain:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="tolerances_selected_grain"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTableWidget" name="tolerances_table">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>175</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>2θ</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>η</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ω</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="ScientificDoubleSpinBox" name="threshold">
       <property name="decimals">
        <number>8</number>
       </property>
       <property name="maximum">
        <double>10000000.000000000000000</double>
       </property>
       <property name="value">
        <double>25.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="use_spots_data_file">
       <property name="text">
        <string>Use Spots Data File</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="threshold_label">
       <property name="text">
        <string>Threshold:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="material"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="material_label">
       <property name="text">
        <string>Selected Material:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="num_hkls_selected">
       <property name="text">
        <string>Number of hkls selected:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="npdiv">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="choose_hkls">
       <property name="text">
        <string>Choose HKLs</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="spots_data_file_label">
         <property name="text">
          <string>Spots data file:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="spots_data_file"/>
       </item>
       <item>
        <widget class="QPushButton" name="select_spots_data_file">
         <property name="text">
          <string>Select File</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vertical_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>view_grains_table</tabstop>
  <tabstop>material</tabstop>
  <tabstop>choose_hkls</tabstop>
  <tabstop>tolerances_selected_grain</tabstop>
  <tabstop>tolerances_table</tabstop>
  <tabstop>npdiv</tabstop>
  <tabstop>threshold</tabstop>
  <tabstop>use_spots_data_file</tabstop>
  <tabstop>spots_data_file</tabstop>
  <tabstop>select_spots_data_file</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>hedm_calibration_options_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>hedm_calibration_options_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
